Dockerfile

```diff
@@ -21,6 +20,9 @@ LABEL org.opencontainers.image.url="https://github.com/docker-mailserver"
 LABEL org.opencontainers.image.documentation="https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md"
 LABEL org.opencontainers.image.source="https://github.com/docker-mailserver/docker-mailserver"
 
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
why did you switch from ARG to ENV

Reply:
ARG is only visible during build time, while ENV will "export" the variable and its value in the running image as well. This is desired because we want to have the vars available during setup (and beyond).

Reply:
Yeah i know the difference. Just wondering why your tz change now has the requirement of setting this as an ENV. Is it dpkg-reconfigure that needs that value? I only know this in combination with apt (which we only need during build time)

Reply:
> This is desired because we want to have the vars available during setup (and beyond).

I cannot think of a reason why we would want to have this var available in the image. BTW we wanted to get rid of almost all ENVs in the Dockerfile. I still guess most of them don't serve any purpose (anymore?). I've that on my to-do.

Reply:
Both of these variables ensure that while building the image there is no interaction required, and the same goes for `dpkg-reconfigure` during setup. I'd leave these two variables in the Dockerfile (as the only ones actually needed), otherwise we may run into problems. If we do want to remove them, I'd need to export the second variable again during debconf and also enable the CLI flags. I'd vote: if you two think it's better to remove them from the Dockerfile, I will do so, otherwise I'd leave it as is now.

Reply:
> Both of these variables ensure that while building the image there is no interaction required

Yes. We were just curious, why you changed from ARG to ENV. Having this var while building is fine (as ARG). But there is no general need to have it in the final image (ENV).
And if apt/dpkg-reconfigure are used, IMO it's best practice, to call these programs with explicit options like `dpkg-reconfigure -f noninteractive`.

Reply:
Alright, then I will take care of it.